woocommerce/woocommerce-ios
# Shipping Labels: handle the value selected in Carrier and Rates

Part of #4085 

## Description
As part of Shipping Labels, I managed the value selected in Carrier and Rates, and now is also possible to re-open the Carrier and Rates screen for selecting a new rate.
In this PR:
- Added the Done button in Carrier and Rates screen. It will be enabled only if there is a rate selected.
- Pressing the Done button in Carrier and Rates screen the rate selected will be shown under the "Carrier and Rates" cell in "Create Shipping Label" view.
- Now it's possible to re-open the Carrier and Rates screen with the previous rate selected, and change it.

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail that is eligible for the shipping label.
3. Press the button "Create Shipping Label".
4. Press the button "Continue" in Create Shipping Label form, under the **Ship From** cell. Confirm the address.
5. Press the button "Continue" in Create Shipping Label form, under the **Ship To** cell. Confirm the address.
6. Press the button "Continue" in Create Shipping Label form, under the **Package Details** cell.
7. You should see the screen that shows the Package Details screen and all the items that will be shipped.
8. Press the "Package selected" row. 
9. Select a package, then press "Done".
10. Add a Total package weight, press "Done".
11. The "Shipping Carrier and Rates" cell should be enabled now. Press it.
12. You should see the new screen "Carrier and Rates". It should display first the ghost cells, then the data fetched from the API.
13. Select a carrier and press Done.
14. The info of the selected carrier and the rate should be shown on the previous screen.
15. You should be able to change the selected rate.


| Cell Shipping Carrier and Rates with info showed            |  Selected carrier and rate |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 12 Pro - 2021-05-26 at 18 12 25](https://user-images.githubusercontent.com/495617/119695082-20af0680-be4e-11eb-8c3b-b6ba828109a2.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-26 at 18 12 34](https://user-images.githubusercontent.com/495617/119695090-23116080-be4e-11eb-88d5-5e2494a5cfc7.png)



Update release notes:
